cli/tests/common/mod.rs

```diff
@@ -160,7 +160,7 @@ impl TestEnvironment {
         cmd
     }
 
-    fn get_ok(&self, mut cmd: assert_cmd::Command) -> (String, String) {
+    pub(crate) fn get_ok(&self, mut cmd: assert_cmd::Command) -> (String, String) {
```

Comment:
nit: you can use `get_stdout_string(&assert)`, etc. instead. (see `test_no_user_configured()` for example.)

I'm not strongly against this change, but the function name is odd as you mentioned before.

Reply:
I’ll look into this.

Reply:
Okay, I’ve undone this change and just inlined what the test needs.